.github/workflows/ci.yml

```diff
@@ -37,7 +37,8 @@ jobs:
         run: deno lint
 
       - name: Typecheck
```

Comment:
Deno v1.20.1 does not seem to support `deno check` yet

https://github.com/denoland/deployctl/blob/d00ce100e65825895dbf17a5850f5402fc641096/.github/workflows/ci.yml#L27